lesson_02/job1/tests/test_storage.py

```diff
@@ -0,0 +1,65 @@
+"""
+Tests api.py
+# TODO: write tests
+"""
+import json
+import os.path
+from unittest import TestCase, mock
+from lesson_02.job1.tests.common import remove_dir
+from lesson_02.job1.storage import save_to_disk
+
+class SaveToDiskTestCase(TestCase):
+    temp_dir = '/tmp/de07/tests/raw/sales'
```

Comment:
you can use this lib for temp dir and temp files: https://docs.python.org/3/library/tempfile.html